Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Date.md

```diff
@@ -20,7 +20,9 @@ The only setting that is available for manipulating the Date property is to set
 
 	@(Model.Content.GetPropertyValue<DateTime>("datePicker").ToString("dd MM yyyy"))
 
-### Dynamic: 
+### Dynamic (Obsolete):
+
+The below example is using Dynamic access content access, which is considered obsolete and is not recommended to use. However the example is included for historical reasons if for instance a developer has overtaken a project where this approach is being used. This approach will be obsolete when Umbraco 8 is released and therefore is best to use the strongly typed example listed above.

```

Comment:
I would prefer to link to the common pitfalls doc rather than repeating this, I would suggest this:

Please see [Common Pitfalls](../../../../Reference/Common-Pitfalls/index.md#dynamics) for information about why the Dynamic approach is obsolete